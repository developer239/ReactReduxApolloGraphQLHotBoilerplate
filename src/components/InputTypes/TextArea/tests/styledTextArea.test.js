import React from 'react'
import { shallow } from 'enzyme'
import { theme } from 'styles'
import { StyledTextArea } from '../index'
import 'jest-styled-components'


describe('StyledTextArea Component', () => {
  it('has correct default styles', () => {
    const wrapper = shallow(
      <StyledTextArea theme={theme} />,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
