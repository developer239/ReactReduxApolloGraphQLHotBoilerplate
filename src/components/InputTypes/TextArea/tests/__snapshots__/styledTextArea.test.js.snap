// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledTextArea Component has correct default styles 1`] = `
.c0 {
  padding: 10px 10px;
  border: 1px solid #ccd1d9;
  margin-top: 10px;
  width: 400px;
  height: 200px;
  background-color: transparent;
}

<textarea
  className="c0"
/>
`;
